docs/release/trg-0/trg-1-4.md

```diff
@@ -0,0 +1,71 @@
+---
+title: TRG 1.04 - Diagrams as Code
+---
+
+| Status | Created       | Post-History          |
+|--------|---------------|-----------------------|
+| Draft  | 20-Sept-2023  | created initial Draft |
+|        |               |                       |
+
+## Why
+
+To have an easier approach to update current static files and have a better overview of the changes within the product pictures.
+
+This helps people to understand our workflows within our products and make it easier to understand the product itself.
+Also updating the static files will be easier and faster to do for every committer or maintainer of our repositories.
+
+## Description
+
+All static files **must** be created with code and also stored as `.svg` file within your product repository.
+After a committers [decision](https://github.com/eclipse-tractusx/sig-infra/discussions/19) there are 2 languages accepted, either [PUML](https://plantuml.com/en/) or [Mermaid](https://mermaid.js.org/).
+These code files (`.puml`,`.mmd`,`mermaid`) and static `.svg` files **must** be stored within the `docs/static` folder.
+
+This includes:
+
+- diagrams
+- flowcharts
+- sequence diagrams
+- state diagrams
+- gantt charts
+- etc.
+
+## PlantUML / Mermaid snippets
+
+### PlantUML example
+
+``` bash
```

Comment:
As far as I know, code blocks with syntax highlighting should look like this (without spaces between the fence and the language):

```suggestion
```bash
```

But why did you specify bash as the language in the first place? If there is no suitable language, I would recommend omitting the language or using `plain`.
The same applies to the example with the mermaid.

Reply:
jep, to syntax highlighting based on [markdown](https://www.markdownguide.org/extended-syntax/#syntax-highlighting) your right, then just a info that this also works also with a blank within GitHub.

``` bash
demo-text with bash and blank
``` 

```bash
demo-text with bash
```

Reply:
updated pr-branch 
https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/401/commits/b8f9f8ff878110ec1781b2f56eafb0474db42d58